src/nrnpython/nrnpy_nrn.cpp

```diff
@@ -1499,17 +1499,17 @@ static PyObject* NPySecObj_insert(NPySecObj* self, PyObject* args) {
         PyObject* tpyobj;
         if (PyArg_ParseTuple(args, "O", &tpyobj)) {
             auto _tpyobj_tracker = nb::borrow(tpyobj);
+            auto self = nb::borrow((PyObject*) raw_self);
             // Returned object to be discarded
-            auto out_o = nb::steal(PyObject_CallMethod(tpyobj, "insert", "O", (PyObject*) self));
+            auto out_o = nb::steal(PyObject_CallMethod(tpyobj, "insert", "O", self.ptr()));
```

Comment:
Now what ever happens in `PyObject_CallMethod` we have a `+1` on the reference count. It can't drop to zero.

Reply:
The perceived advantage is that I don't need to know anything about `PyObject_CallMethod`.

(I agree that a well-mannered `"insert"` is unlikely to DECREF the `self`; but I don't know how to cleanly rule out that it's impossible.)